src/Sai2Model.h

```diff
@@ -866,6 +924,14 @@ class Sai2Model {
 MatrixXd matrixRangeBasis(const MatrixXd& matrix,
 						  const double& tolerance = 1e-3);
 
+/**
+ * @brief Computes the svd data for a matrix
```

Comment:
More details in the comment. Explain that it is a thin svd, what it means in terms of the number of rows and columns of the matrices U and V, the fact that S is a vector and not a diagonal matrix, the size of S, and which computation recovers the original matrix (It would be something like U * s.asDiagonal() * V.transpose() )